lib/utils/util.js

```diff
@@ -33,12 +33,7 @@ export function toURI(path:string):Uri {
 }
 
 export function hasFlowPragma(content:string) {
-	const hasPragma =
-		content.startsWith('/* @flow */') ||
-		content.startsWith('// @flow\n') ||
-		/^\s*\/\*+\s*@flow\s*\*+\//m.test(content) ||
-		/^\s*\/\/\s*@flow\s*$/m.test(content)
-	return hasPragma
+	return /^\s*(\/*\*+|\/\/)\s*@flow/m.test(content);
```

Comment:
Previous regex didn't catch things like
```js
/*
 * @flow
 */
```
which are pretty common.
I tried to cover most cases out there: http://regexr.com/3fumm

We could further optimise checking for pragma on larger files while typing, by caching the result and revalidating if the pragma is still there once in a while.